src/tests/run.sh

```diff
@@ -11,7 +11,7 @@ function print_usage {
     echo 'Optional arguments:'
     echo '  -h|--help                        : Show usage information.'
     echo '  -v, --verbose                    : Show output from each test.'
-    echo '  <arch>                           : One of x64, x86, arm, arm64, loongarch64, riscv64, wasm. Defaults to current architecture.'
```

Comment:
https://github.com/dotnet/runtime/pull/74470/files#diff-cee66bb1ca705ee11c9869ce5c1709f67374ff9b979ecfce36265545d941c152L70-L72

Here is not the current architecture.

If keep the comment, should modify the default architecture as `__arch=$CPUName`.

Reply:
Hi, @am11 
Could you help reviewing this PR?
Thanks

Reply:
The comment looks good. To avoid code duplication, you can replace lines 42-82  with:

```sh
################################################################################
# Handle Arguments
################################################################################

source "$repoRootDir/eng/native/init-os-and-arch.sh"
ARCH=$arch
```

Reply:
Ok, thanks